library/src/lib/toggle/toggle.component.ts

```diff
@@ -15,13 +23,33 @@ export class ToggleComponent implements OnInit {
     disabled: boolean = false;
 
     @Input()
+    id: string;
+
     checked: boolean = false;
 
-    @Input()
-    id: string;
+    onChange: any = () => {};
+    onTouched: any = () => {};
+
+    get inputText() {
+        return this.checked;
+    }
+
+    set inputText(value) {
```

Comment:
Maybe a bit confusing to have get and set InputText on a toggle?

Reply:
fixed